src/backend/wiki_game_info/wiki_game_info.ts

```diff
@@ -50,19 +50,21 @@ export async function getWikiGameInfo(
         isMac ? getInfoFromAppleGamingWiki(title) : null
       ])
 
-    const steamID = gamesdb?.steamID ? gamesdb.steamID : ''
-    const [protondb, steamdeck] = await Promise.all([
-      getInfoFromProtonDB(isLinux ? steamID : ''),
-      getSteamDeckComp(isLinux ? steamID : '')
-    ])
+    let steamInfo = null
+    if (isLinux) {
+      const steamID = pcgamingwiki?.steamID ?? gamesdb?.steamID
```

Comment:
If there's a new default and the gamesdb is a fallback, I think the fallback should be tested as well.